pkg/model/stack_serializer.go

```diff
@@ -373,6 +375,20 @@ func (p *PortRaw) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+func IsSkippablePort(port int32) bool {
+	relationalDBPortList := []int32{7210, 3306, 1521, 1830, 5432, 1433, 3050}
```

Comment:
Let's use a map instead of a list so it makes the check operation a constant time operation.